pkg/storegateway/series_refs.go

```diff
@@ -64,36 +71,41 @@ func (m seriesChunkRef) Compare(other seriesChunkRef) int {
 	return 0
 }
 
-// seriesChunkRefsIterator implements an interator over seriesChunkRefsSet.
-type seriesChunkRefsIterator struct {
+// seriesChunkRefsIteratorImpl implements an iterator returning a sequence of seriesChunkRefs.
+type seriesChunkRefsIteratorImpl struct {
```

Comment:
Note to reviewers: I couldn't come up with a better name so far. `seriesChunkRefsIterator` is already taken in the interface.